src/openclsim/plot/step_chart.py

```diff
@@ -5,8 +5,17 @@
 from .log_dataframe import get_log_dataframe
 
 
-def get_step_chart(simulation_objects):
-    """Get the step chart of the container levels."""
+def get_step_chart(simulation_objects, container_map={"default":""}):
```

Comment:
Do not use dictionaries as arguments, but use None and instantiate in __init__

Reply:
ok